stumpy/Mmotifs.py

```diff
@@ -0,0 +1,297 @@
+import numpy as np
+import logging
+import stumpy
+import matplotlib.pyplot as plt
+from . import core
+from . import config
+from stumpy.core import _preprocess
+
+logger = logging.getLogger(__name__)
+
+
+def Mmotifs(
+    T: np.ndarray,
+    P: np.ndarray,
+    I: np.ndarray,
+    max_matches: int = 10,
+    max_motifs: int = 1,
+    cutoff: np.ndarray = None,
+    visualize_mdl: bool = False,

```

Comment:
Please remove `visualize_mdl`. Instead, you may store a list of `mdl` and `subspaces` that are computed and return it back to the user at the end if it is relevant